src/docs/Components/Accordion/index.mdx

```diff
@@ -46,17 +46,31 @@ The accordion pattern can be used with an illustrative icon before the title whe
 
 <Preview path="icon" nude/>
 
-## Alignment
+## Variations
+
+### Sizes
+
+You will find two different sizes for the accordions. The smallest version should be used for secondary content. The default version is bigger and is best suited for readable content.
+
+<Preview path="sizes" nude/>
+
+<Highlight theme="warning" title="Be aware">
+
+Always think about accessibility when you use smaller text content sizes.
```

Comment:
~~when you use smaller text content sizes~~ when you use the smaller accordion sizes